NationStatesAPIBot/NationStatesAPIBot/Manager/NationManager.cs

```diff
@@ -30,6 +40,22 @@ public static int GetNationCountByStatusName(string name)
             }
         }
 
+        public static async Task<bool> IsNationPendingSkippedSendOrFailedAsync(string name)
+        {
+            using(var dbContext = new BotDbContext(_config))
+            {
+                var nation = await dbContext.Nations.Include(n => n.Status).FirstOrDefaultAsync
+                    (
+                    n => n.Name == name 
+                    && n.Status.Name == "pending" 
+                    && n.Status.Name == "skipped" 
+                    && n.Status.Name == "send" 
+                    && n.Status.Name == "failed"
+                    );
+                return nation != null;
+            }
+        }
+
```

Comment:
Method name says "Or", but you're using &&?

Reply:
Just found and fixed by myself :)